packages/xwayland.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Xwayland < Package
+  description 'X server configured to (only) work with weston'
+  homepage 'https://x.org'
+  version '1.19.3'
+  source_url 'https://www.x.org/releases/individual/xserver/xorg-server-1.19.3.tar.gz'
+  source_sha256 '8f93b98f1ac9fbd87515bfe329a069b48bbec98e5329584ab5fbf759a0953b8d'
+
+  depends_on 'libepoxy'
+  depends_on 'xorg_proto'
+  depends_on 'libxtrans'
+  depends_on 'libxkbfile'
+  depends_on 'wayland'
+  depends_on 'eudev'
+  depends_on 'libxfont'
+  depends_on 'libbsd'
+  depends_on 'nettle'
+  depends_on 'libtirpc'
+  depends_on 'pixman'
+
+  def self.build
+    system "./configure --prefix=#{CREW_PREFIX} --disable-docs --disable-devel-docs --enable-xwayland --disable-xorg --disable-xvfb --disable-xnest --disable-xquartz --disable-xwin"
```

Comment:
Why are we disabling so many X configure options here?

Reply:
Yes, they are not needed here, as we don't need the full X-Server, just the bare Wayland stuff to get Weston working.

Reply:
OK, cool.  Just curious.

Reply:
I noticed there was one file in `/usr/local/lib` so we probably need `--libdir=#{CREW_LIB_PREFIX}` in this package as well.  See below:
```
$ crew files xwayland
(i) xwayland: X server configured to (only) work with weston
https://x.org
version 1.19.3
/usr/local/bin/Xwayland
/usr/local/lib/xorg/protocol.txt
/usr/local/share/man/man1/Xserver.1
/usr/local/share/X11/xkb/compiled/README.compiled
Total found: 4
```